src/crocoddyl_reaching.cpp

```diff
@@ -37,15 +37,15 @@
 
 namespace panda_torque_mpc
 {
-    CrocoddylReaching::CrocoddylReaching(pin::Model _model_pin, const boost::shared_ptr<pin::GeometryModel>& _collision_model ,
+    CrocoddylReaching::CrocoddylReaching(pin::Model model_pin, const boost::shared_ptr<pin::GeometryModel>& collision_model ,
```

Comment:
```suggestion
    CrocoddylReaching::CrocoddylReaching(const pin::Model model_pin, const boost::shared_ptr<pin::GeometryModel>& collision_model ,
```
`mode_pin` can be constant as it is not modified in here